accounting/print_hierarchy.py

```diff
@@ -28,9 +29,17 @@ def print_full_hierarchy(conn):
     dataframe = pd.read_sql_query(select_stmt, conn)
 
     if len(dataframe) == 0:
-        raise Exception("No root bank found")
+        try:
```

Comment:
i'm a little confused by this design pattern.  why capture the exception here.  why not capture the exception when `print_full_hierarchy()` is called?

Reply:
Are you thinking to instead capture the exception in **accounting_cli.py**?

```python
elif args.func == "print_hierarchy":
    print(ph.print_full_hierarchy(conn))
```

I guess my reasoning was I was capturing exceptions from inside the subcommands themselves, not the program (**accounting_cli.py**) that calls those functions, so I tried to make it consistent. 

Does it make more sense to capture the exception in **accounting_cli.py**?

Reply:
ok.  In that case, perhaps I'd suggest;

```
try:
  if ...:
     raise ...
  elif ...:
     raise ...
except:
   print(e)
   sys.exit(1)
```

so no need for the double try/exception.  I think that double try/except is what confused me the most.

Reply:
Ah, okay. Thanks. Sorry for the confusion. That change should be pushed up now.